controllers/replica_set_controller.go

```diff
@@ -561,6 +630,14 @@ func buildStatefulSetModificationFunction(mdb mdbv1.MongoDBCommunity) statefulse
 	)
 }
 
+func buildArbitersModificationFunction(mdb mdbv1.MongoDBCommunity) statefulset.Modification {
+	return statefulset.Apply(
+		statefulset.WithReplicas(int(mdb.Spec.Arbiters)),
```

Comment:
this should be using the function which takes scaling into account right? Rather than spec.Arbiters directly.